src/main/java/it/gov/pagopa/payhub/activities/dto/IngestionFlowFileDTO.java

```diff
@@ -32,4 +34,10 @@ public class IngestionFlowFileDTO implements Serializable {
     private Long pdfGenerated;
     private String codRequestToken;
     private String codError;
+    private String pspIdentifier;
+    private Date flowDateTime;
+    private BigInteger state;
```

Comment:
this is the status? if yes, I expect it should be a String or better an enum (actually could be String, we could introduce the enum after)